MANIFEST.in

```diff
@@ -0,0 +1 @@
+include pylettize/palettes/*
```

Comment:
I've seen people add the README and LICENSE here. Should we do that too?

Reply:
Should probably be included